python/lsst/ts/observing/utilities/auxtel/latiss/getters.py

```diff
@@ -51,10 +51,7 @@ async def get_image(
             logger.debug("Image grabbed and ISR performed successfully")
             return exp
 
-        # TODO: DM-34120 -- will allow the use of proper exception
-        # handling below
-        # except dafButler.exceptions.InconsistentDataIdError:
-        except Exception as err:
+        except ValueError as err:
```

Comment:
Consider replacing the double `logger.warning` + `logger.debug` by a single `logger.exception`. Then you can do `except ValueError:` here (dropping the `as err`).

If you don't want an exception being logged at every iteration, then maybe remove the warning as well?

Reply:
Ya, that can be cleaned up since now we only catch the one type of error. When we were catching all errors (due to the butler issue) it was useful for understanding what was happening.
Fix implemented.